### DISCUSSION TITLE
Bubble Release in Liquid Metal

### DISCUSSION THREAD

**Discussion by astephan023** [idaholab/moose#26874] [Q&A Modules: General]
I am trying to run simulations for a UO2 fuel pellet in a liquid metal bonded fuel pin. As the fission gasses are released they will build up on the surface of the fuel pellet until the bubble is big enough and is released. I'm having trouble finding a physics model for simulating the bubbles releasing after they reach a given size. Has anyone worked with something like this or know which module would be best used for this problem?

TIA,
Adam

**Comment by GiudGiud** [comment]
Hello

We are adding limited 2 phase capabilities. I dont think they will be enough for your problem but feel free to check them out. 
It s a mixture model and it s hosted in the Navier Stokes module. 
See the inputs in `modules/navier_stokes/test/tests/finite_volume/two_phase/mixture_model`

Guillaume
